crates/iceberg/src/spec/manifest.rs

```diff
@@ -966,6 +966,12 @@ impl ManifestEntry {
         self.sequence_number
     }
 
+    /// File sequence number.
+    #[inline]
+    pub fn file_sequence_number(&self) -> Option<i64> {
+        self.file_sequence_number
+    }
```

Comment:
The `file_sequence_number` relies on this value but there was no way to get at the `ManifestEntry::file_sequence_number` before.